greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExportRun.java

```diff
@@ -0,0 +1,155 @@
+package com.greenback.kit.model;
+
+import java.time.Instant;
+
+public class AutoExportRun extends GreenbackObject{
+    private String referenceId;
+    private AutoExportRunType type;
+    private ProcessingStatus status;
+    private String userId;
+    private Long typeTagId;
+    private String lockedBy;
+    private String hostname;
+    private Instant killedAt;
+    private Instant completedAt;
+    private Double progress;
+    private String message;
+    private AutoExportRequest parameters;
+    private String resultsDoc;
```

Comment:
This wouldn't be a string, it would be the results, and should be a nice object.